decentraland/unity-renderer
# Chore: social improvements

## What does this PR change?

Fix #3997
Fix #4170 
...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/social-improvements
2. Verify that all points listed in #3997 and #4170 are correctly applied

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
